pyramid/paster.py

```diff
@@ -53,7 +53,7 @@ def get_appsettings(config_uri, name=None, options=None, appconfig=appconfig):
         global_conf=options)
 
 def setup_logging(config_uri, fileConfig=fileConfig,
-                  configparser=configparser):
+                  configparser=configparser, global_conf=None):
```

Comment:
OK. Push some new commits to handle these issues:

- 	Change arg order to setup_logging …			6e52898
-	docs/api/paster.rst: Add global_conf arg to setup_logging			e9ac5ad
-	Document new global_conf arg to setup_logging			5bc0669